go.mod

```diff
@@ -1,18 +1,18 @@
 module sigs.k8s.io/cluster-api-provider-vsphere
 
-go 1.19
+go 1.20
```

Comment:
I don't know enough about CAPV yet, but this usually requires bumping images used in Prow. Let's check

Reply:
Valid point. Checked in test-infra:

The used images provide go 1.19.9.

So I reverted the go.mod version change for now. Let's keep this separately.

Reply:
Can you open a follow-up issue to track this, please?